spec/schemas/Uint32.yml

```diff
@@ -0,0 +1,4 @@
+type: integer
+format: int64
```

Comment:
Why int64 format? would 'int32' format + minimum: 0 works

Reply:
int32 generates `int` field in java, the biggest number is 2,147,483,647 which is lower than the allowed uint32 4,294,967,295. So we need to go up in the possible type, in this case, int6,  uint32 doesn't exist in open API. This is not a problem in Javascript because there is not `int` type, just long `number`.

Check the java sdk issue.